### DISCUSSION TITLE
My thoughts on the email-typo functionality.

### DISCUSSION THREAD

**Discussion by Warrobot10** [monkeytypegame/monkeytype#4454] [Ideas] [answered]
Instead of having a list full of possible typos, it would be much more efficient to have a list of correct email domains.
Would love to work on this change, but don't really know what to change.

**Comment by Miodec** [comment, marked as answer]
I don't agree. The list of "correct" domains is infinite, because people can self host emails. The list of common typos for popular domains is smaller.
